Plugins/Statistics/GitImpact/ImpactControl.cs

```diff
@@ -362,7 +362,7 @@ private void UpdatePathsAndLabels()
                     h_max = Math.Max(h_max, y);
 
                     // Add week date label
-                    _weekLabels.Add((new PointF(x + (BlockWidth / 2), y), weekDate));
+                    _weekLabels.Add((new PointF(x + (BlockWidth / 2f), y), weekDate));
```

Comment:
Ditto.

Reply:
Skipping ImpactControl (a not much used plugin, looks OK to me), just WaitSpinner.

This change puts the small circles closer to the "outer circle". The cos/sin part should make the fraction differ some anyway, this will make no difference.
This is what it looks like after the change:
![image](https://user-images.githubusercontent.com/6248932/47321071-cb863500-d653-11e8-9292-6b1f5d935773.png)

To closer see that images cannot be put on center I used radius 3, the small circles are fuzzy:
![image](https://user-images.githubusercontent.com/6248932/47320647-52d2a900-d652-11e8-9bd7-09079bfc14a6.png)

The difference to the original (correct size 8) is very small, (you cannot really see that the circles are offset), but they look the same as after the change:
![image](https://user-images.githubusercontent.com/6248932/47320742-97f6db00-d652-11e8-9f77-e386f9e94b59.png)

A better way is to align the middle of the small circle to a border: 
![image](https://user-images.githubusercontent.com/6248932/47320975-5dda0900-d653-11e8-9cce-b0d5c44a8a8c.png)

`                e.Graphics.FillEllipse(
                    _brushes[i],
                    (int)Math.Round((_centre.X - (_dotRadius / 2f)) + (_circleRadius * angle.cos)),
                    (int)Math.Round((_centre.Y - (_dotRadius / 2f)) + (_circleRadius * angle.sin)),
                    _dotRadius,
                    _dotRadius);
`

I suggest this is kept as is in the PR